tests/console/lshw.pm

```diff
@@ -23,7 +23,9 @@ sub run {
 
     # Check various output formats, -sanitize is used to not spill test machine serial numbers into public
     # On some architectures fields like "product" or "vendor" or section "*-pci" might not exist, so trying a common base.
-    validate_script_output("lshw -sanitize", sub { m/description.*\*-memory\n.*\*-network/s });
+    assert_script_run("lshw -sanitize | grep -A5 'description'");
+    assert_script_run("lshw -sanitize | grep -A5 '\\*-memory\$'");
+    assert_script_run("lshw -sanitize | grep -A5 '\\*-network'");
```

Comment:
Why did you split it up?

If you use single quotes, you have  to do less escaping.

Reply:
ok. thx for the pointer that validat_script_output() not working. We should take a look to it. But not withhin this PR.

Sure keep it.